index.html

```diff
@@ -96,8 +96,7 @@ <h4 class="icon">LET’S CONNECT</h4>
       </section>
     </main>
 
-
-    <section class="scroll" id="portfolio">
+    <section class="projects" id="portfolio">
```

Comment:
[OPTIONAL]The main goal of this project is to remove the projects section from the HTML file and generate it dynamically with JavaScript. In your HTML file, there should be only an empty container to target it from your JavaScript selector, and the rest need to be generated dynamically as you did in the modal. Unfortunately, previous reviewer @BregornOriginal missed this point.